docs/governance/how-is-etherlink-governed.md

```diff
@@ -20,7 +20,7 @@ It consists of three periods: a Proposal period and a Promotion period, which ar
 
 ### 1. Proposal period
 
-During this period, Etherlink bakers have about 3.5 days to submit and upvote proposals for new kernels.
+During this period, Etherlink bakers have about 3 days to submit and upvote proposals for new kernels.
```

Comment:
imho we should specify that this should be determined by a number of L1 blocks, and then provide the approximate length in days.

The question however is on how we want to proceed.

So we fix to future values, or current values?

Reply:
Could we fix with future values, but put in a warning block or something with temporary information about current values ?

Reply:
Ok. So these are not the current values then